cmd/dashboards/pods/pods.json

```diff
@@ -384,6 +412,84 @@
       "transform": "table",
       "type": "table"
     },
+    {
+      "columns": [],
+      "datasource": "k8s-cluster",
+      "fontSize": "100%",
+      "gridPos": {
+        "h": 4,
+        "w": 24,
+        "x": 0,
+        "y": 7
+      },
+      "id": 23,
+      "links": [],
+      "pageSize": 10,
+      "scroll": true,
+      "showHeader": true,
+      "sort": {
+        "col": 0,
+        "desc": true
+      },
+      "styles": [
+        {
+          "alias": "",
+          "colorMode": null,
+          "colors": [
+            "rgba(245, 54, 54, 0.9)",
+            "rgba(237, 129, 40, 0.89)",
+            "rgba(50, 172, 45, 0.97)"
+          ],
+          "decimals": 2,
+          "pattern": "/Time/",
+          "thresholds": [],
+          "type": "hidden",
+          "unit": "short"
+        },
+        {
+          "alias": "restart",
+          "colorMode": null,
+          "colors": [
+            "rgba(245, 54, 54, 0.9)",
+            "rgba(237, 129, 40, 0.89)",
+            "rgba(50, 172, 45, 0.97)"
+          ],
+          "dateFormat": "YYYY-MM-DD HH:mm:ss",
+          "decimals": 2,
+          "mappingType": 2,
+          "pattern": "Value",
+          "rangeMaps": [
+            {
+              "from": "-1",
+              "text": "stoped",
+              "to": "0"
+            },
+            {
+              "from": "0",
+              "text": "running",
+              "to": "100"
+            }
+          ],
+          "thresholds": [],
+          "type": "number",
+          "unit": "short"
+        }
+      ],
+      "targets": [
+        {
+          "expr": "sum(kube_pod_container_status_restarts_total{namespace=\"$namespace\", pod=\"$pod\"}) by (pod, container)",
```

Comment:
Can we put this into the "Pod Image Version" graph and perhaps rename it to "Pod Container Status"?

Reply:
Also, I think this graph should be moved to "Pod Info" panel. Currently, they are in the "Service Info" panel which is weird.